From: Stefan Hajnoczi <stefanha@gmail.com>
Date: Sun, 11 Nov 2012 13:52:26 +0100
Subject: [054/106] staging: line6: drop unused CONFIG_LINE6_USB_DEBUG

commit bf83e30ec654e3a6d79e89ae70d1c9811a2387df upstream.

The CONFIG_LINE6_USB_DEBUG option is no longer relevant since dyndbg
dev_dbg() is now used instead of a compile-time decision whether to
enable debug messages or not.

Signed-off-by: Stefan Hajnoczi <stefanha@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/Kconfig |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/staging/line6/Kconfig b/drivers/staging/line6/Kconfig
index 43120ff..a5ded12 100644
--- a/drivers/staging/line6/Kconfig
+++ b/drivers/staging/line6/Kconfig
@@ -23,14 +23,6 @@ menuconfig LINE6_USB
 
 if LINE6_USB
 
-config LINE6_USB_DEBUG
-	bool "print debug messages"
-	default n
-	help
-	  Say Y here to write debug messages to the syslog.
-
-	  If unsure, say N.
-
 config LINE6_USB_DUMP_CTRL
 	bool "dump control messages"
 	default n
